.pre-commit-config.yaml

```
@@ -95,7 +95,7 @@ repos:
 
 - repo: https://github.com/PyCQA/flake8.git
   # NOTE: This is kept at v4 for until WPS starts supporting flake v5.
-  rev: 4.0.1  # enforce-version: 4.0.1
+  rev: 6.0.0  # enforce-version: 4.0.1
```

This entry must remain unchanged but pre-commit.ci has no way of knowing that:
```suggestion
  rev: 4.0.1  # enforce-version: 4.0.1
```